File: src/main/java/xyz/acrylicstyle/hackReport/utils/PlayerInfo.java
package xyz.acrylicstyle.hackReport.utils;

import java.util.UUID;

public class PlayerInfo {
    private final String name;
    private final UUID uuid;
    private int reports = 0;
    private int kills = 0;
    private int deaths = 0;

    public PlayerInfo(String name, UUID uuid) {
        this.name = name;
        this.uuid = uuid;
    }

    public String getName() {
        return name;
    }

    public UUID getUniqueId() {
        return uuid;
    }

    public int getReports() {
        return reports;
    }

    public void setReports(int reports) {
        this.reports = reports;
    }

    public void increaseReports() {
        this.reports++;
    }

    public void increaseKills() {
        this.kills++;
    }

    public int getKills() {
        return kills;
    }

    public void increaseDeaths() {
        this.deaths++;
    }

    public int getDeaths() {
        return deaths;
    }
}

File: src/main/java/xyz/acrylicstyle/hackReport/utils/PlayerInfo.kt
package xyz.acrylicstyle.hackReport.utils

import java.util.UUID

class PlayerInfo(val name: String, val uniqueId: UUID) {
    var reports = 0
    var kills = 0
        private set
    var deaths = 0
        private set

    fun increaseReports() {
        reports++
    }

    fun increaseKills() {
        kills++
    }

    fun increaseDeaths() {
        deaths++
    }
}